carp.deployments.core/src/commonTest/kotlin/dk/cachet/carp/deployments/application/ParticipationServiceTest.kt

```diff
@@ -76,6 +76,42 @@ interface ParticipationServiceTest
         assertEquals( setOf( expectedAssignedDevice ), retrievedInvitation.assignedDevices )
     }
 
+    @Test
+    fun getActiveParticipationInvitations_with_multiple_participation_succeeds() = runTest {
+        val (participationService, deploymentService, accountService) = createSUT()
+        val protocol = createSinglePrimaryDeviceProtocol()
```

Comment:
It's best for integration tests to try to reflect real-world scenarios. Especially, given that these tests are also used to output _example JSON requests_ which are included in releases (generated through `OutputParticipationServiceTestRequests`, but added manually), and these output JSON requests in turn are used for regression testing when API versions are bumped to test API migrations (`BackwardsCompatibilityTest`).

It's not very common/expected (though possible) to have a study protocol with a single device, which is assigned to both a "father" and "mother". Instead, we could make this test more representative of a multi-device scenario, which in addition also increases code coverage on top of what is already tested in the test above.